### PR TITLE
elastic: security bump to 6.8.22/7.16.2

### PR DESCRIPTION
**Pull request by hydrapolic**
log4j 2.17.0 update

### CODE REVIEW

File: app-admin/logstash-bin/logstash-bin-7.16.2.ebuild [hunk 1]
```diff
@@ -32,9 +32,9 @@ src_prepare() {
 	default
 
 	local d
-	for d in aarch64-Linux arm-Linux Darwin i386-Linux i386-SunOS \
+	for d in aarch64-FreeBSD aarch64-Linux arm-Linux Darwin i386-Linux i386-SunOS \
 		i386-Windows mips64el-Linux ppc64-AIX ppc64le-Linux ppc64-Linux \
-		ppc-AIX sparcv9-Linux sparcv9-SunOS x86_64-DragonFlyBSD \
+		ppc-AIX s390x-Linux sparcv9-Linux sparcv9-SunOS x86_64-DragonFlyBSD \
```
Comment (thesamesam) [inline app-admin/logstash-bin/logstash-bin-7.16.2.ebuild:37]: nice

File: www-apps/kibana-bin/kibana-bin-7.16.2.ebuild [hunk 2]
```diff
@@ -81,7 +81,7 @@ src_install() {
 
 pkg_postinst() {
 	elog "This version of Kibana is compatible with Elasticsearch $(ver_cut 1-2) and"
-	elog "Node.js 14. Some plugins may fail with other versions of Node.js (Bug #656008)."
+	elog "Node.js 16. Some plugins may fail with other versions of Node.js (Bug #656008)."
```
Comment (thesamesam) [inline www-apps/kibana-bin/kibana-bin-7.16.2.ebuild:84]: Sorry I'd missed the earlier PR for this. I'd again opened the tab and it got lost!

File: app-admin/logstash-bin/logstash-bin-7.16.2.ebuild (hunk 1 above)
Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-7.16.2.ebuild:37]: What do you mean by that?

File: www-apps/kibana-bin/kibana-bin-7.16.2.ebuild (hunk 2 above)
Comment (hydrapolic) [reply www-apps/kibana-bin/kibana-bin-7.16.2.ebuild:84]: No problem :)

**Comment by hydrapolic** [comment]
Thank you @thesamesam

File: app-admin/logstash-bin/logstash-bin-7.16.2.ebuild (hunk 1 above)
Comment (thesamesam) [reply app-admin/logstash-bin/logstash-bin-7.16.2.ebuild:37]: Just well spotted that upstream gained support for it!

Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-7.16.2.ebuild:37]: Actually it's pretty harmless, just installs a file that won't be used. But it was in the portage output log and Ago opened a bug for it, so why not fixing it when it's Xmas time :)

**Pull request closed without merging**